thoth/storages/graph/postgres.py

```diff
@@ -3287,14 +3289,30 @@ def get_python_package_versions_all_count(
         if python_version is not None:
             query = query.filter(PythonPackageVersion.python_version == python_version)
 
+        if sort_by:
+            if sort_by == QuerySortTypeEnum.PACKAGE_VERSION.value:
+                query = query.order_by(func.count(
+                            PythonPackageVersion.package_version).desc()
+                        )
+
+            elif sort_by == QuerySortTypeEnum.PACKAGE_NAME.value:
+                query = query.order_by(PythonPackageVersion.package_name)
+
+            else:
+                raise SortTypeQueryError(
+                        f"The sort key known in Thoth for queries are {[e.value for e in QuerySortTypeEnum]!r}"
+                        )
+
+        groups_count = query.count()
```

Comment:
Will this perform another query to the database?

Reply:
Yes it will

Reply:
IMHO it would be better to reuse `_count` methods - for API simplicity and clearness.

Reply:
we don't have a `_count` for the groups created by a query using `group_by` method inside, we need a new one :)